_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title:  "Book Review: The Intelligent Investor"
+date:   2019-02-24 22:40:05
+categories: jekyll update
+---
+
+### Background
+Around 2014 after being recommended this book by numerous sources I picked it up and gave it a read. Great books profoundly influence the way you think for the rest of your life. The Intelligent Investor is such a book. It challenged my initial ideas on investment, how to think about assets, and how to think about wealth. Contained within it are sound principles for first preserving what one has and then increase that pile of money safely with time. Benjamin Graham gave me the tools to navigate the waters of finance without making the big mistakes. This is not a book for how to make a quick buck from bitcoin or the next craze or fad. This is a book on careful and conservative thinking in making returns while considering many factors for oneself.
+
+### Disclaimer and Basics on Retirement
+
+I am simply an enthusiastic lay person
+who happens to know some math, a fair bit of personal finance,
+and likes reading books on the topic of investing. Please do your own research
+and don't ask me for specific advice about any companies,
+markets, etc.
+
+**I will not help you with individual financial stuff. Please do not ask.** Consult with a fiduciary advisor with any specific question you have and make sure your 401k company is not screwing you over. *Hint*: They are probably screwing you over, watch [The Retirement Gamble](https://www.pbs.org/video/frontline-retirement-gamble/).
+
+
+### The Gist
+
+_The Intelligent Investor_ and _Security Analysis_ are the only investment books I ever needed to read, all the others were further elaboration on Benjamin Graham's points. I decided to do this book review because his principles to preserve wealth, reducing the amount of false positives, and truly invest for the long run <sup>1</sup> are timeless techniques. For the purposes of brevity, I will focus on what Benjamin Graham calls a _Defensive Investor_ instead of the _Enterprising Investor_.
+
+Intelligent investing is not about being the smartest person in the room, it is about following sound principles to their ends. An intelligent investor is someone who develops temperance, patience, and control. She is learned, meticulous in their reasoning, and defensive in their thinking. She becomes familiar with the major ideas in many fields related to investing. These fields include, but are not limited to: psychology, history, basic statistics, meditation, and human evolution. An intelligent investor has more in common with an endurance athlete than a superstar sports player, in this game consistency can beat talent in many cases.
```

Comment:
She is learned, meticulous in **her** reasoning, and defensive in **her** thinking.